ui/src/components/StatusIcon.js

```diff
@@ -1,27 +1,102 @@
 import React from 'react';
 
 import Icon from 'material-ui/Icon';
+
+import { CircularProgress } from 'material-ui/Progress';
 import Tooltip from 'material-ui/Tooltip';
 
+import green from 'material-ui/colors/green';
+import yellow from 'material-ui/colors/yellow';
+import red from 'material-ui/colors/red';
+
+
+const statusColorIconMap = {
```

Comment:
We recently added `Stopping`/`Stopped`/`Starting` statuses, along with /stop and /start endpoints. See swagger:

https://notebooks.firecloud.org/#/cluster
https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/swagger/api-docs.yaml#L470

I wonder if it makes sense to expose this in the UI?

Reply:
It certainly does. It's already exposed through the `"Cannot parse " + status` message, but having useful icons is certainly better. I've pushed changes to add that.